test/acceptance/create_webdriver.robot

```diff
@@ -15,38 +15,54 @@ Create Webdriver Creates Functioning WebDriver
 
 Create Webdriver With Bad Driver Name
     [Documentation]    Invalid browser name
-    Run Keyword And Expect Error    'Fireox' is not a valid WebDriver name
-    ...    Create Webdriver    Fireox
+    Run Keyword And Expect Error    'Fireox' is not a valid WebDriver name    Create Webdriver
+    ...    Fireox
 
 Create Webdriver With Duplicate Arguments
     [Documentation]    Invalid values in arguments
     ${kwargs}=    Create Dictionary    arg=1
-    Run Keyword And Expect Error    Got multiple values for argument 'arg'.
-    ...    Create Webdriver    Firefox    kwargs=${kwargs}    arg=2
+    Run Keyword And Expect Error    Got multiple values for argument 'arg'.    Create Webdriver
+    ...    Firefox    kwargs=${kwargs}    arg=2
 
 Create Webdriver With Bad Keyword Argument Dictionary
     [Documentation]    Invalid arguments types
-    Run Keyword And Expect Error    kwargs must be a dictionary.
-    ...    Create Webdriver    Firefox    kwargs={'spam': 'eggs'}
+    Run Keyword And Expect Error    kwargs must be a dictionary.    Create Webdriver    Firefox
+    ...    kwargs={'spam': 'eggs'}
+
+Get Browser Info
```

Comment:
Could you write cleaner tests and writing more tests would be useful. At least: 
1) Clean the not required logging
2) Write tests that verifies something
3) Also it would be good idea to test when Open Browser keyword is used.
4) I think we do not use selenium grid, but at least give it some thought.

The 3&4 are pretty much extra because it is always the WebDriver driver object giving the return value.